.expeditor/verify_private.pipeline.yml

```diff
@@ -114,12 +114,15 @@ steps:
             "/go/src/github.com/chef/automate"
           ]
 
-  - label: ":semgrep: Published"
+  - label: ":semgrep: Security"
     command:
     - echo "running in $(pwd)"
     - export SEMGREP_JOB_URL=$BUILDKITE_BUILD_URL
     - export SEMGREP_BRANCH=$BUILDKITE_BRANCH
-    - python -m semgrep_agent --publish-token "\$SEMGREP_TOKEN" --publish-deployment \$SEMGREP_ID
+    - export SEMGREP_REPO_NAME=chef/automate
+    - echo "branch=[$BUILDKITE_BRANCH], PR base branch=[$BUILDKITE_PULL_REQUEST_BASE_BRANCH]"
+    - \[ -z "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" \] && echo "manual build; using 'master' as base branch"
+    - python -m semgrep_agent --publish-token "\$SEMGREP_TOKEN" --publish-deployment \$SEMGREP_ID --baseline-ref \${BUILDKITE_PULL_REQUEST_BASE_BRANCH:-master}
```

Comment:
The `--baseline-ref` parameter activates the differential scan mode of semgrep, so it only examines changes in each PR rather than the whole codebase.